manifest.webapp

```diff
@@ -45,7 +45,7 @@
       "verbs": ["ALL"]
     },
     "contacts": {
-      "description": "Required for the service to update the myself contact",
+      "description": "Required for the service to update the myself contact, and to add contacts to contracts/accounts",
```

Comment:
Wondering if you should not update the en & fr.json files too

Reply:
For what ?

Reply:
There is no mention of this permission in `en.json` or `fr.json` since the home is preinstalled.

Reply:
I see what you mean. But we have some context where the home is not installed by default (for now on those contexts we can not install it from the store I think). 

This it not part of this PR, but I think we should not rely on "this app has, for now, a specific behavior" to not do what the cozy's ecosystem wants (not sure if it's clear enough)

Reply:
Yes I agree. Thanks for having created the issue :)